embed-block-figma.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
- * Plugin Name:       Figma Block
- * Plugin URI:        https://github.com/10up/figma-block
+ * Plugin Name:       Embed Block for Figma
+ * Plugin URI:        https://github.com/10up/embed-block-figma
```

Comment:
Are we going to update the GitHub repo? Right now that is set to https://github.com/10up/figma-block/ and so this URL won't work

Reply:
Yes, was waiting to confirm the changes here were complete before changing the repo